examples/custom_usage_counter/custom_usage_counter.py

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env python3
 import asyncio
-from dataclasses import dataclass
-from datetime import datetime, timedelta
-from decimal import Decimal
 import pathlib
 import sys
+from datetime import datetime, timedelta
+from decimal import Decimal
+
+from dataclasses import dataclass
```

Comment:
`dataclasses` are stdlib since 3.7. and should be in the same grupe as `datetime`, I wonder if checks on 3.7-3.10 will pass now :thinking: